src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -66,6 +67,7 @@
             <Setter Property="Background" Value="Transparent"/>
             <Setter Property="BorderThickness" Value="0"/>
             <Setter Property="SelectionMode" Value="Single"/>
+            <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Disabled"/>
```

Comment:
I noticed that for taint, some locations have a very long text, which causes the "navigability warning" to be less visible.
I removed the scroll so the text would be wrapped.

![image](https://user-images.githubusercontent.com/60586879/105359024-38a02080-5bf7-11eb-8c1f-db7e175f1c05.png)